### PR TITLE
Optimise `box`/`package` icons

### PR DESCRIPTION
**Pull request by danielbayley**
Related to `box-dashed` and `box-transparent` in #1344.

`package` alts:

preview

### CODE REVIEW

**Review by jguddas — APPROVED** [review]

File: icons/package-check.svg [hunk 1]
```diff
@@ -11,7 +11,7 @@
 >
   <path d="m16 16 2 2 4-4" />
   <path d="M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
-  <path d="M16.5 9.4 7.55 4.24" />
+  <path d="m7.5 4.3 9 5.1" />
```
Comment (karsa-mistmere) [inline icons/package-check.svg:14]: I think we could do with an extra point of precision here as usual:
```suggestion
  <path d="m7.5 4.27 9 5.15" />
```

File: icons/package-minus.svg [hunk 2]
```diff
@@ -11,7 +11,7 @@
 >
   <path d="M16 16h6" />
   <path d="M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
-  <path d="M16.5 9.4 7.55 4.24" />
+  <path d="m7.5 4.3 9 5.1" />
```
Comment (karsa-mistmere) [inline icons/package-minus.svg:14]:
```suggestion
  <path d="m7.5 4.27 9 5.15" />
```

File: icons/package-plus.svg [hunk 3]
```diff
@@ -12,7 +12,7 @@
   <path d="M16 16h6" />
   <path d="M19 13v6" />
   <path d="M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
-  <path d="M16.5 9.4 7.55 4.24" />
+  <path d="m7.5 4.3 9 5.1" />
```
Comment (karsa-mistmere) [inline icons/package-plus.svg:15]:
```suggestion
  <path d="m7.5 4.27 9 5.15" />
```

File: icons/package-search.svg [hunk 4]
```diff
@@ -10,7 +10,7 @@
   stroke-linejoin="round"
 >
   <path d="M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
-  <path d="M16.5 9.4 7.55 4.24" />
+  <path d="m7.5 4.3 9 5.1" />
```
Comment (karsa-mistmere) [inline icons/package-search.svg:13]:
```suggestion
  <path d="m7.5 4.27 9 5.15" />
```

File: icons/package-x.svg [hunk 5]
```diff
@@ -10,7 +10,7 @@
   stroke-linejoin="round"
 >
   <path d="M21 10V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l2-1.14" />
-  <path d="M16.5 9.4 7.55 4.24" />
+  <path d="m7.5 4.3 9 5.1" />
```
Comment (karsa-mistmere) [inline icons/package-x.svg:13]:
```suggestion
  <path d="m7.5 4.27 9 5.15" />
```

File: icons/package.svg [hunk 6]
```diff
@@ -9,8 +9,8 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M16.5 9.4 7.55 4.24" />
-  <path d="M21 16V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l7-4A2 2 0 0 0 21 16z" />
-  <polyline points="3.29 7 12 12 20.71 7" />
-  <line x1="12" x2="12" y1="22" y2="12" />
+  <path d="m7.5 4.3 9 5.1" />
```
Comment (karsa-mistmere) [inline icons/package.svg:12]:
```suggestion
  <path d="m7.5 4.27 9 5.15" />
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Comment by danielbayley** [comment]
@karsa-mistmere `commit`ted…

What do we thing of extending the tape down the side? https://github.com/lucide-icons/lucide/pull/1481#issue-1838147006 cc: @jguddas @ericfennis

**Comment by jguddas** [comment]
> What do we thing of extending the tape down the side?

:-1: KISS

**Comment by ericfennis** [comment]
Nope, lets keep it simple!

**Review by ericfennis — APPROVED** [review]

**Pull request merged**